spec/search_builders/hyrax/admin_admin_set_member_search_builder_spec.rb

```diff
@@ -13,7 +13,17 @@
 
     it 'searches for valid work types' do
       expect(builder.filter_models(solr_params))
-        .to contain_exactly("{!terms f=has_model_ssim}Monograph,#{Hyrax.config.collection_class}")
+        .to contain_exactly(include("{!terms f=has_model_ssim}Monograph"))
```

Comment:
I see what you are doing here.  This won't fail if more collections are added to the list.  Are you ok with that?  Testing for them all in one test will let us know if another one gets added.  This would most likely be ok, but we might want to know so we can evaluate the impact.  Not a blocker for me if this is the way you want to go.

Reply:
> This won't fail if more collections are added to the list. Are you ok with that?

i think so. this approach would have made this test less fragile to the changes that caused the issue here to begin with. i see that as an improvement